tests/acceptance/test_mender-connect.py

```diff
@@ -193,14 +202,18 @@ def test_mender_connect_auth_changes(
         """Test that mender-connect can re-establish the connection on D-Bus signals"""
 
         try:
-            dbus_set_token_and_url(connection, "token1", "http://localhost:5000")
```

Comment:
I think this we should keep as it was. The test scenario here is that the server is up and running already when `mender-connect` starts.

Reply:
Okey, since the tests fail due to timeout when we first set the token and url, and then let mender-connect start. But is successful when we do it after mender-connect has started. I will investigate the issue :+1: